configure.ac

```diff
@@ -217,18 +217,11 @@ AC_PROG_CXX
 if test -z "$CXX"; then
     AC_MSG_ERROR([A c++ compiler is required to build BZFlag])
 fi
-AX_CXX_COMPILE_STDCXX_0X
-if test "x$ax_cv_cxx_compile_cxx0x_native" != xyes; then
-    if test "x$ax_cv_cxx_compile_cxx0x_cxx" = xyes; then
-	CXXFLAGS="$CXXFLAGS -std=c++0x"
-	user_CXXFLAGS="$user_CXXFLAGS -std=c++0x"	# for --enable-debug
-    elif test "x$ax_cv_cxx_compile_cxx0x_gxx" = xyes; then
-	CXXFLAGS="$CXXFLAGS -std=gnu++0x"
-	user_CXXFLAGS="$user_CXXFLAGS -std=gnu++0x"	# for --enable-debug
-    else
-	AC_MSG_ERROR([A c++ compiler with C++0x support is required to build BZFlag])
-    fi
-fi
+
+# C++14 is the minimum standard. C++17 isn't widely available
+#AX_CXX_COMPILE_STDCXX([11], [noext])
```

Comment:
If we're moving to C++14 being the new minimum, do we need C++11 commented out?

Reply:
I left it for comparison but I’m happy to remove it.

Reply:
I don't feel strongly either way about this, I was more so curious about its purpose

Reply:
It was there in the first place because I wanted to verify that it compiled that way (C++11) vs the old config as a sanity check. As evidenced by the branch name, I intended to go all the way to C++17, but I found the Mac compiler (at least) has insufficient C++17 support). At one time I had flags for all 3 versions. As I tested, I dropped down to just the one. Say the word and this comment is gone.